tests/testZogy.py

```diff
@@ -0,0 +1,209 @@
+#
+# LSST Data Management System
+# Copyright 2016-2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+from __future__ import absolute_import, division, print_function
+import unittest
+
+from builtins import range
+from past.builtins import basestring
+import numpy as np
+
+import lsst.utils.tests
+import lsst.afw.image as afwImage
+#import lsst.afw.geom as afwGeom
+import lsst.afw.math as afwMath
+#import lsst.meas.algorithms as measAlg
+import lsst.daf.base as dafBase
+import testImageDecorrelation as testID  # for makeFakeImages
+
+from lsst.ip.diffim.zogy import (ZogyTask, ZogyConfig, ZogyMapReduceConfig)
+from lsst.ip.diffim.imageMapReduce import ImageMapReduceTask
+
+try:
+    type(verbose)
+except NameError:
+    verbose = False
+
+
+def setup_module(module):
+    lsst.utils.tests.init()
+
+
+class ZogyTest(lsst.utils.tests.TestCase):
+    """A test case for the Zogy task.
+    """
+
+    def setUp(self):
+        self.psf1_sigma = 3.3  # sigma of psf of science image
+        self.psf2_sigma = 2.2  # sigma of psf of template image
+
+        self.statsControl = afwMath.StatisticsControl()
+        self.statsControl.setNumSigmaClip(3.)
+        self.statsControl.setNumIter(3)
+        self.statsControl.setAndMask(afwImage.MaskU.getPlaneBitMask(["INTRP", "EDGE", "SAT", "CR",
+                                                                     "DETECTED", "BAD",
+                                                                     "NO_DATA", "DETECTED_NEGATIVE"]))
+
+    def _setUpImages(self, svar=100., tvar=100., varyPsf=0.):
+        """Generate a fake aligned template and science image.
+        """
+        self.svar = svar  # variance of noise in science image
+        self.tvar = tvar  # variance of noise in template image
+
+        self.im1ex, self.im2ex \
+            = testID.makeFakeImages(svar=self.svar, tvar=self.tvar,
+                                    psf1=self.psf1_sigma, psf2=self.psf2_sigma,
+                                    n_sources=10, psf_yvary_factor=varyPsf,
+                                    seed=666, verbose=False)
+        # Create an array corresponding to the "expected" subtraction (noise only)
+        np.random.seed(666)
```

Comment:
Since you're setting this seed and the keyword seed on line 74 above to the same value, you should define `seed = 666` as a variable and reference that in both places.